pkgs/development/python-modules/lockfile/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs
```

Comment:
typically we don't pass in `pkgs`, only `lib`.

Reply:
fixed